08.ls_object/lib/ls/file_information.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
+
+require 'date'
+require 'etc'
+
+module Ls
+  class FileInformation
```

Comment:
-l オプションありの時のために使うクラスなので

```suggestion
  class LongFormatInformation
```

の方が適している気がする